app/src/main/java/org/simple/clinic/home/overdue/OverdueScreen.kt

```diff
@@ -297,6 +297,11 @@ class OverdueScreen : BaseScreen<
     overdueProgressBar.visibility = View.GONE
   }
 
+  override fun handleNoOverduePatientsViewVisibility(shouldShowEmptyView: Boolean) {
```

Comment:
I am not sure about this. Usually we handle these visibility check in UI renderer. We end up toggling `overdueRecyclerView` visibility here and `showOverdueAppointments` function as well.

Maybe we can instead move the conditional check in UI renderer and toggle between `showOverdueAppointments` and ` hideOverdueAppointment`? That way we can have separate functions for handling no overdue appointments view as well. Like `showNoOverdueAppointmentsView` and `hideNoOverdueAppointmentsView`. WDYT?